.circleci/config.yml

```diff
@@ -1,557 +0,0 @@
-#
-# Defaults
-#
-
-# shared
-
-defaults: &defaults
-  working_directory: ~/italia-app
-
-node_cache_key: &node_cache_key v1.17-dependencies-{{ checksum "yarn.lock" }}
-
-android_sdk_cache_key: &android_sdk_cache_key v1-android-sdkmanager-packages-{{ checksum "scripts/circleci-android-setup.sh" }}
-
-android_jars_cache_key: &android_jars_cache_key jars-{{ checksum "android/build.gradle" }}-{{ checksum  "android/app/build.gradle" }}
-
-ios_gems_cache_key: &ios_gems_cache_key ruby-{{ checksum "Gemfile.lock" }}
-
-release_tag: &release_tag /^[0-9]+\.[0-9]+\.[0-9]+(-rc\.[0-9]+)?$/
-
-# nodejs builds
-defaults_js: &defaults_js
-  <<: *defaults
-  docker:
-    - image: cimg/node:16.19.0
-
-# android builds
-defaults_android: &defaults_android
-  <<: *defaults
-  docker:
-    - image: cimg/android:2022.09.2-node
-  environment:
-    TERM: "dumb"
-    ANDROID_SDK_BUILD_TOOLS_REVISION: "29.0.3"
-    ANDROID_SDK_BUILD_API_LEVEL: "30"
-    ANDROID_GOOGLE_API_LEVEL: "23"
```

Comment:
This are no more necessary in GA because the action `setup-android` download the right once looking at the `build.gradle` file?

Reply:
The action, looking at it's repo, just installs the command line tools (see [here](https://github.com/android-actions/setup-android/blob/7c5672355aaa8fde5f97a91aa9a99616d1ace6bc/src/main.ts#L12)) without any possibility to specify the API level of the SDK, so I assume it would fetch specifications from `gradle` file